ci-operator/config/red-hat-storage/ocs-ci/README.md

```diff
@@ -0,0 +1,73 @@
+# Openshift ODF Interop Interoperability Tests<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [General Information](#general-information)
+- [Purpose](#purpose)
+- [Process](#process)
+  - [Cluster Provisioning and Deprovisioning: `firewatch-ipi-aws`](#cluster-provisioning-and-deprovisioning-firewatch-ipi-aws)
+  - [Test Setup, Execution, and Reporting Results - Openshift GitOps-interop-aws](#test-setup-execution-and-reporting-results---openshift-gitops-interop-aws)
+    - [AWS Cluster configuration](#aws-cluster-configuration)
+- [Prerequisite(s)](#prerequisites)
+  - [Environment Variables](#environment-variables)
+
+## General Information
+
+- **Repository**: [RedHatStorage/ocs-ci](https://github.com/red-hat-storage/ocs-ci)
+- **Operator Tested**: [OCS-CI](https://ocs-ci.readthedocs.io/en/latest/)
+- **Maintainers**: RedHatStorage
+
+## Purpose
+
+To provision the necessary infrastructure and using that infrastructure to execute Openshift ODF interop tests.
+The results of these tests will be reported to the appropriate sources following execution.
+
+## Process
+
+The Openshift ODF Interop scenario can be broken into the following basic steps:
+
+1. Provision a test cluster on AWS
+2. Select and label worker nodes for ODF
+2. Install the ODF operator on the `openshift-storage` namespace
+3. Setup StorageCluster for ODF named `ocs-storagecluster`
+3. Execute tests and archive results
+4. Deprovision a test cluster.
+
+### Cluster Provisioning and Deprovisioning: `firewatch-ipi-aws`
+
+Please see the [`firewatch-ipi-aws`](https://steps.ci.openshift.org/workflow/firewatch-ipi-aws) documentation for more information on this workflow. This workflow is not maintained by the Interop QE team.
+
+### Test Setup, Execution, and Reporting Results
+
+#### AWS Cluster configuration
+
+For ODF, we are required to use these specifications: 
+
+- `COMPUTE_NODE_TYPE`: m5.4xlarge 
+- `CONTROL_PLANE_INSTANCE_TYPE`: m5.4xlarge
```

Comment:
can we leave this empty, so that the OCP installer will use the default type? In our AWS IPI deployment we don't specify the instance type for control plane nodes - https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/aws/ipi_3az_rhcos_3m_3w.yaml

Reply:
Trying this approach

Reply:
@ebenahar Jobs passed, updated the scenario.